tests/test_mlip_calculators.py

```diff
@@ -123,16 +128,32 @@ def test_extra_mlips(architecture, device, kwargs):
     "kwargs",
     [
         {
-            "model_path": MODEL_PATH / "v5.27.2024/best_model.pt",
-            "model": MODEL_PATH / "v5.27.2024/best_model.pt",
+            "architecture": "alignn",
+            "model_path": MODEL_PATH / "v5.27.2024" / "best_model.pt",
+            "model": MODEL_PATH / "v5.27.2024" / "best_model.pt",
+        },
+        {
+            "architecture": "alignn",
+            "model_path": MODEL_PATH / "v5.27.2024" / "best_model.pt",
+            "path": MODEL_PATH / "v5.27.2024" / "best_model.pt",
+        },
+        {
+            "architecture": "sevennet",
+            "model_path": SEVENNET_PATH,
+            "path": SEVENNET_PATH,
+        },
+        {
+            "architecture": "sevennet",
+            "model_path": SEVENNET_PATH,
+            "model": SEVENNET_PATH,
         },
         {
-            "model_path": MODEL_PATH / "v5.27.2024/best_model.pt",
-            "path": MODEL_PATH / "v5.27.2024/best_model.pt",
+            "architecture": "sevennet",
+            "model_path": "",
```

Comment:
Maybe worth adding a test without model_path as a kwarg at all?

Reply:
that will fail... which part of the code will test?

Reply:
Fail as in throw the wrong error, or not throw an error?

I mean we cuold check not specifying `model_path` (or similar) also throws an error, which is different to setting it to an invalid value (`""`), which throws a potentially different error.

Reply:
mute point now since labels are supported.